From bcc21950c72823cf2895ed91efc39cefc3f5038a Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 21 Jun 2018 15:33:31 +0800
Subject: [PATCH 4057/5242] MLK-18652 ARM: dts: imx6: update ARM sw2iso timing
 setting

commit  fffd973ea64113db777bef95ba8bf6c253141a32 from
https://source.codeaurora.org/external/imx/linux-imx.git

The sw2iso count should cover ARM LDO ramp-up time,
the MAX ARM LDO ramp-up time may be up to more than
100us, this patch sets sw2iso to 0xf (~384us) which
is the default value.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-14x14-arm2.dts        |    2 --
 arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts   |    2 +-
 arch/arm/boot/dts/imx6ul-14x14-evk.dtsi        |    2 +-
 arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts |    4 ++--
 arch/arm/boot/dts/imx6ul-9x9-evk.dts           |    2 +-
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts  |    2 +-
 arch/arm/boot/dts/imx6ull-9x9-evk.dts          |    2 +-
 7 files changed, 7 insertions(+), 9 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-14x14-arm2.dts b/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
index 3a04d28..896748c 100644
--- a/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-14x14-arm2.dts
@@ -206,8 +206,6 @@
 };
 
 &gpc {
-	fsl,cpu_pupscr_sw2iso = <0x2>;
-	fsl,cpu_pupscr_sw = <0x1>;
 	fsl,cpu_pdnscr_iso2sw = <0x1>;
 	fsl,cpu_pdnscr_iso = <0x1>;
 	fsl,ldo-bypass = <1>; /* use ldo-bypass, u-boot will check it and configure */
diff --git a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
index ee19df6..e79ab43 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
@@ -188,7 +188,7 @@
 };
 
 &gpc {
-	fsl,cpu_pupscr_sw2iso = <0x1>;
+	fsl,cpu_pupscr_sw2iso = <0xf>;
 	fsl,cpu_pupscr_sw = <0x0>;
 	fsl,cpu_pdnscr_iso2sw = <0x1>;
 	fsl,cpu_pdnscr_iso = <0x1>;
diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
index 45040f7..066d0bf 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
@@ -253,7 +253,7 @@
 };
 
 &gpc {
-	fsl,cpu_pupscr_sw2iso = <0x1>;
+	fsl,cpu_pupscr_sw2iso = <0xf>;
 	fsl,cpu_pupscr_sw = <0x0>;
 	fsl,cpu_pdnscr_iso2sw = <0x1>;
 	fsl,cpu_pdnscr_iso = <0x1>;
diff --git a/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
index 94f5d86..26ae733 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-lpddr2-arm2.dts
@@ -182,8 +182,8 @@
 };
 
 &gpc {
-	fsl,cpu_pupscr_sw2iso = <0x2>;
-	fsl,cpu_pupscr_sw = <0x1>;
+	fsl,cpu_pupscr_sw2iso = <0xf>;
+	fsl,cpu_pupscr_sw = <0x0>;
 	fsl,cpu_pdnscr_iso2sw = <0x1>;
 	fsl,cpu_pdnscr_iso = <0x1>;
 	fsl,ldo-bypass = <1>; /* use ldo-bypass, u-boot will check it and configure */
diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk.dts b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
index 29896ea..caa2ed5 100644
--- a/arch/arm/boot/dts/imx6ul-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
@@ -242,7 +242,7 @@
 };
 
 &gpc {
-	fsl,cpu_pupscr_sw2iso = <0x1>;
+	fsl,cpu_pupscr_sw2iso = <0xf>;
 	fsl,cpu_pupscr_sw = <0x0>;
 	fsl,cpu_pdnscr_iso2sw = <0x1>;
 	fsl,cpu_pdnscr_iso = <0x1>;
diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index 63ed0d1..292e72d 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -203,7 +203,7 @@
 };
 
 &gpc {
-	fsl,cpu_pupscr_sw2iso = <0x1>;
+	fsl,cpu_pupscr_sw2iso = <0xf>;
 	fsl,cpu_pupscr_sw = <0x0>;
 	fsl,cpu_pdnscr_iso2sw = <0x1>;
 	fsl,cpu_pdnscr_iso = <0x1>;
diff --git a/arch/arm/boot/dts/imx6ull-9x9-evk.dts b/arch/arm/boot/dts/imx6ull-9x9-evk.dts
index 8f7c6f8..9991169 100644
--- a/arch/arm/boot/dts/imx6ull-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ull-9x9-evk.dts
@@ -242,7 +242,7 @@
 };
 
 &gpc {
-	fsl,cpu_pupscr_sw2iso = <0x1>;
+	fsl,cpu_pupscr_sw2iso = <0xf>;
 	fsl,cpu_pupscr_sw = <0x0>;
 	fsl,cpu_pdnscr_iso2sw = <0x1>;
 	fsl,cpu_pdnscr_iso = <0x1>;
-- 
1.7.9.5

